### ISSUE TITLE
IBM plugin installation instructions do not work for terraform version 13

### ISSUE THREAD

**Issue by ErikEvenson** [IBM-Cloud/terraform-provider-ibm#1787]
Terraform version 13 introduced new plugin installation requirements:

https://www.terraform.io/upgrade-guides/0-13.html#new-filesystem-layout-for-local-copies-of-providers

```
% terraform init

Initializing the backend...

Initializing provider plugins...
- Finding latest version of hashicorp/ibm...

Error: Failed to install provider

Error while installing hashicorp/ibm: provider registry registry.terraform.io
does not have a provider named registry.terraform.io/hashicorp/ibm
```

**Comment by l2fprod** [comment]
here are instructions that work for 
```
Terraform v0.13.0
+ provider localdomain/provider/ibm v1.10.0
```

https://stackoverflow.com/questions/63365156/how-to-use-the-ibm-cloud-provider-plug-in-for-terraform-with-terraform-0-13

**Comment by hkantare** [comment]
yes we are working on the process of registering the provider...We are working on preparing the provider for registration
https://www.terraform.io/docs/registry/providers/publishing.html#preparing-your-provider

**Comment by mkumatag** [comment]
Instructions didn't work completely! still hitting with the following error:

```
Initializing modules...

Initializing the backend...

Initializing provider plugins...
- Using previously-installed localdomain/provider/ibm v1.10.0
- Using previously-installed hashicorp/null v2.1.2
- Finding latest version of hashicorp/ibm...

Error: Failed to install provider

Error while installing hashicorp/ibm: provider registry registry.terraform.io
does not have a provider named registry.terraform.io/hashicorp/ibm

Error: terraform.Apply failed: failed to initialize Terraform
```

**Comment by mindjiver** [comment]
Since the provider is now properly published in the upstream repository the following worked for me:

```
terraform {
  required_version = ">= 0.13"
  required_providers {
    ibm = {
      source  = "ibm-cloud/ibm"
      version = "1.11.2"
    }
  }
}
```

**Comment by SionArtingstall** [comment]
Using the following provider as per docs here https://registry.terraform.io/providers/IBM-Cloud/ibm/1.13.1 fails

```
terraform {
  required_providers {
    ibm = {
      source = "IBM-Cloud/ibm"
      version = "1.13.1"
    }
  }
}

provider "ibm" {
  # Configuration options
}
```

Error is:

```

Initializing provider plugins...
- Finding ibm-cloud/ibm versions matching "1.13.0, 1.14.0, 1.13.0"...

Error: Failed to query available provider packages

Could not retrieve the list of available versions for provider ibm-cloud/ibm:
no available releases match the given constraints 1.13.0, 1.14.0, 1.13.0
```

**Comment by hkantare** [comment]
We now registerd with hasicorp 
```
terraform {
  required_providers {
    ibm = {
      source = "IBM-Cloud/ibm"
      version = "1.16.1"
    }
  }
}

provider "ibm" {
  # Configuration options
}
```
